contents/handbook/people/hiring-process.md

```diff
@@ -4,25 +4,43 @@ sidebar: Handbook
 showTitle: true
 ---
 
-This page will walk you through how we hire at PostHog. The goal is to have a lightweight process that optimizes for speed for the candidate, but above all for quality of the hire.
+This page will walk you through how we hire at PostHog. The goal is to have a lightweight process that optimizes for speed and quality of the hire, but also ensures the best possible candidate experience! 
 
-## Deciding to Hire
+## Our Hiring Process in short
 
-Every hire introduces complexity to the organisation and increases our burn rate. As a result, we think very carefully about each new role, and we set an extremely high bar for the people that we do hire. 
+For most roles, our hiring process looks like this: 
```

Comment:
I think there's a conflict of interest here where the Handbook is primarily for internal use but inevitably makes its way into the laptops of the candidates. I do like the sentiment of making this a nicer read for candidates but I wonder if this is indeed the primary objective. I guess this is more of a reminder to ourselves than anything else?

But I don't actually have the answer. @jamesefhawkins thoughts?

Reply:
Agreed - they're for different audiences.

I think the best move here is probably in the careers section of the website to have stuff that outlines the process written for candidates.

Reply:
From the feedback I get from candidates is that they get lost in our handbook, and they know they read information somewhere, but can't find it later. Also the navigation from the careers page (and back) to the handbook isn't super easy. 

We have short explainer on the careers page, which links to the handbook. I get that the handbook should be written for an internal audience, but it does create some confusion with candidates, so I felt like it might make sense to have a short version for candidates at the top (given the page is quite big), and a more detailed one for us internally at the bottom. 

This problem might go away with the new careers page, so we can leave it as it is (minus the changes to the superday), if people feel like we should keep the handbook entirely internal and accept that it might not be a great read from candidates.

Reply:
> From the feedback I get from candidates is that they get lost in our handbook, and they know they read information somewhere, but can't find it later

Definitely think this is valid and something we should solve. A quick fix is - create the version you'd want a candidate to see, from their perspective, and we'll add to the nav here:

<img width="1227" alt="Screenshot 2021-02-03 at 14 55 05" src="https://user-images.githubusercontent.com/47497682/106764619-dbad5d00-662f-11eb-9b79-70f251398503.png">

Reply:
(and then can update any links in the posthog.com/careers main text to the candidate-facing version)

Reply:
That could be really cool actually. I'll see if I can build that and will also check with Cory to see what time frame we have for the careers page, so I don't do any unnecessary work

Reply:
@eltjehelene shall we cancel this change then to close the issue, then build something along those lines?

Reply:
Yes, that's ok. I'll close it